cloud-media-livestream/keypublisher/README.md

```diff
@@ -0,0 +1,302 @@
+# User-managed encryption key publisher
```

Comment:
Could you label the instructions with numbers? It's a bit hard to follow the different texts and when to read onto a part of the instruction or if it spills into the next section.

Reply:
I've added numbering for the H2 headers so that each general section is easier to spot. Please let me know if you think the H3s also need numbering.

Reply:
Looks much better! I think numbered H2 headers are easier to work with than numbered H3 headers (or some mix).